README.md

```diff
@@ -38,6 +38,33 @@ For each kind of action (link or form), you have to add a data-toggle="ajax" to
     </a>
 If the link contains the data-form property, it will submit the form using the properties of the link.
 
+### Auto-refresh a form
+
+You can automatically send an ajax request to update your form when an input/select change, just add the data attribute "data-refreshonchange":
+    
+```html
+    <select name="category" data-refreshonchange="true">
+        <option value="transport">Transport</option>
+        <option value="structure">Stucture</option>
+    </select>
+```
+
+Then in your controller's action:
+```php
+$noValidate = $request->query->get('novalidate', false);
+if ($noValidate === false && $form->isValid()) {
+// Form is valid
+}
+if ($request->isXmlHttpRequest() && $noValidate === true ) {
+    return new JsonResponse([
+        'html' => $this->renderView()//render your form 
+    ]);
+}
+```
+
+For some reason you can't refresh part of your form for example for input="file". You can take part from the old html and transfer to the new html, just by add the 
```

Comment:
For some reason, you would not refresh some parts of your form (for example an input type="file". Then, add the data attribute 'data-ignoreonchange="$some_unique_id"'.